tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/matters.controller.js

```diff
@@ -633,7 +634,7 @@
                         vm.IsDupliacteDocument = true; //ToDo:Set it to false on mail upload dialog open
                         vm.IsNonIdenticalContent = false;
 
-                        var selectedOverwriteConfiguration = vm.uploadMessages.overwrite_Config_Property.trim().toLocaleUpperCase(),
+                        var selectedOverwriteConfiguration = vm.globalSettings.overwriteDupliacteFileNameWithDateTimeFor.trim().toLocaleUpperCase(),

```

Comment:
Instead of reading the value from overwrite_Config_Property, now reading from overwriteDupliacteFileNameWithDateTimeFor